tests/020-module.test.js

```diff
@@ -0,0 +1,33 @@
+const mock = require('mock-fs');
+const serverModule = require('../');
+
+describe('Test module export', () => {
+    it('should return an error object because no file', async () => {
+        // mock file in temporary file system
+        mock({
+            '/proc/': {
+                'mdstat': null
+            }
+        });
+        // apply module function
+        const res = await serverModule();
+        expect(res).toEqual({error: 'kernel module not loaded'});
+    });
+
+    /* it('should return an error object because not parsable file', async () => {
+        // mock file in temporary file system
+        mock({
+            '/proc/': {
+                'mdstat': '\n'
+            }
+        });
+
+        // apply module function
+        const res = await serverModule();
+        expect(res).toHaveProperty('error', /Please report the issue to module author!/);
```

Comment:
Same thing as above. This, in my opinion, should log a suiting message to the console server-side, but otherwise throw leading to a `HTTP 500` getting shown in the client which is reserved for a wrong server-side configuration and automatically shows the suiting error messages. It is then for the admin to go through the logs on server-side to look into the problem (and nothing we need in the API response)...

cf. https://github.com/server-state/specs/blob/master/api/routes.md#apiv1module-name -> Possible Status Codes -> `500`

Reply:
Ok, That's a good point.

Reply:
Going with my comment https://github.com/server-state/linux-raid-module/pull/4#discussion_r323403697, this should also just throw with the error message. It is then up to the consumer (`server-base`) to decide how to handle it (this might also be configurable as in which of the options mentioned in my comment get used)...